.github/workflows/rubyonrails.yml

```
@@ -31,7 +31,7 @@ jobs:
         uses: actions/checkout@v3
       # Add or replace dependency steps here
       - name: Install Ruby and gems
-        uses: ruby/setup-ruby@0a29871fe2b0200a17a4497bae54fe5df0d973aa # v1.115.3
+        uses: ruby/setup-ruby@v1.152.0
```

Update to a version of the Action that is aware of Ruby v3.1.4's existence.

Alternatively, this could go with the Action maintainer's [emphatic recommendation](https://github.com/ruby/setup-ruby#versioning) to ride the `v1` tag to always run the latest version of the Action:

```suggestion
        uses: ruby/setup-ruby@v1
```